### PR TITLE
expression: implement vectorized evaluation for `builtinInetNtoaSig`

### PR DESCRIPTION
**Pull request by Reminiscent**
### What problem does this PR solve? 
Implement vectorized evaluation for builtinInetNtoaSig.

### What is changed and how it works?
```
BenchmarkVectorizedBuiltinFunc/builtinInetNtoaSig-VecBuiltinFunc-4                300000              5391 ns/op
BenchmarkVectorizedBuiltinFunc/builtinInetNtoaSig-NonVecBuiltinFunc-4             100000             18383 ns/op
```

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: expression/builtin_miscellaneous_vec.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expression
+
+import (
+	"encoding/binary"
```
Comment (qw4990) [inline expression/builtin_miscellaneous_vec.go]: Please format these packages.

Comment (Reminiscent) [reply expression/builtin_miscellaneous_vec.go]: update. PTAL

File: expression/builtin_miscellaneous_vec.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+	result.ReserveString(n)
+	i64s := buf.Int64s()
+	for i := 0; i < n; i++ {
+		val := i64s[i]
+		if buf.IsNull(i) || val < 0 || uint64(val) > math.MaxUint32 {
+			result.AppendNull()
+			continue
+		}
+		ip := make(net.IP, net.IPv4len)
```
Comment (SunRunAway) [inline expression/builtin_miscellaneous_vec.go]: Could we reuse this buffer during for loop?

Comment (Reminiscent) [reply expression/builtin_miscellaneous_vec.go]: update. PTAL

Comment (SunRunAway) [reply expression/builtin_miscellaneous_vec.go]: Any benchmark result update?

Comment (Reminiscent) [reply expression/builtin_miscellaneous_vec.go]: Update. But it seems that the promotion is not obvious.

Comment (SunRunAway) [reply expression/builtin_miscellaneous_vec.go]: We can use option `benchmem` to see the benchmark of memory.
```
go test -benchmem
```

**Pull request merged**